modin/core/execution/ray/common/deferred_execution.py

```diff
@@ -29,13 +29,20 @@
 
 import pandas
 import ray
+import ray.exceptions
```

Comment:
It seems that a lot of the changes in this file are not directly affected by this pull request and therefore it would be great to move them into a separate pull request.